### ISSUE TITLE
Greek Warrior Helmet Sign

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11573]
**Preferred term label:**
Greek Warrior Helmet Sign

**Synonyms**

**Definition (free text, please give PubMed ID)**
The main prenatal finding is strikingly thick prefrontal edema which when visualized with 3D sonography shows the classical postnatal profile resembling a Greek warrior helmet. PMID 16258951

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:194190

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay 
Adding new term: 
- ID: HP:6001478
- label: Greek warrior helmet sign
- parents: [Fetal head anomaly (HP:0034245)](https://hpo.jax.org/app/browse/term/HP:0034245)
- pmid: PMID:17324995|PMID:16258951|PMID:15305099
- def: A prenatal sonographic finding characterized by a high forehead, prominent glabella, and a broad, un-tapered nasal bridge that merges directly into the brow line. This appearance is said to resemble a Greek (Corinthian) warrior helmet because the wide, solid vertical column down the center of the face is said to resemble the fixed, protective nose guard of a Corinthian helmet.
- comment: This sign is suggestive of the Wolf-Hirschhorn (4p-) syndrome.
- orcid: orcid:0000-0001-9947-035X
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11573
- synonyms:
